FW/src/App.cpp

```diff
@@ -27,6 +27,7 @@ void App::Run() {
     m_stateMachine.Start();
     m_tickTimer.Start();
     m_watch.Button.Start();
+    m_watch.Accelerometer.Start();
```

Comment:
How about adding a BCDWatch::Start() that takes care of starting the tasks it contains? The App class and the BCDWatch class are being quite tightly coupled here.

Reply:
Good idea, since the initialization is handled in the main task function, before the loop is started, the Start() functions more or less replace the init functions.

The BCDWatch clas shas two purposes,

- Collect pin initialization in one place

- Provide some kind of application context for the state classes to access things like debug, RTC and Display

Perhaps it would be nicer to have a header with pin defines and add the application context part to the App-class. Then each driver could take care of initialization of its HAL components, and just get the correct pin mapping from the header.